charts/generic-prometheus-alerts/ci/tests/ingress-alerts-tests-hourly.yaml

```diff
@@ -0,0 +1,45 @@
+---
+rule_files:
+  - ../test-application-ingress.yaml
+
+evaluation_interval: 1h
```

Comment:
put in a separate file from the standard ingress tests since need to evaluate every hour instead of every minute as success responses evaluated over a much longer time period